File: 2951. Find the Peaks.dart
class Solution {
  List<int> findPeaks(List<int> mountain) {
     final List<int> result = [];
     for(var i=1;i<mountain.length-1;i++){
         if(mountain[i-1]<mountain[i] && mountain[i+1]<mountain[i]){
             result.add(i);
         }
     }
     return result;
  }
}

File: 2951. Find the Peaks.ts
function findPeaks(mountain: number[]): number[] {
    let result = [];
    for(let i=1;i<mountain.length-1;i++){
        if(mountain[i-1]<mountain[i] && mountain[i+1]<mountain[i]){
            result.push(i)
        }
    }
    return result
};
